CONTRIBUTORS.md

```diff
@@ -46,6 +48,7 @@ Contributors to the Chapel 1.25.0 release
 * Max Tursi, [HPE]
 * Karlon West, [HPE]
 * Tong Xu, individual contributor
+* anonymous GitHub user @krwmilz
```

Comment:
@krwmilz : I put this in as a placeholder due to your contribution of PR #18669.  Is there a proper name I could use for you instead?  Thanks!

Reply:
Hi Brad, yes you can use Kyle Milz.

This may sound strange but I'm interested in the Programming Language Engineer position advertised at https://chapel-lang.org/jobs.html. I applied through the normal channel months ago but have not got a response. I have professional experience developing C/C++ static analysis tools, and have created my own C/C++ source code rewriter using LLVM. I'm wondering if there is a better way for me to apply?

Reply:
Thanks for the quick response, Kyle (and for the code contribution itself), I'll make that update.

I tend to only get involved in the hiring process pretty late in the game for a given applicant,
so can't comment on your specific case or even the state of the hiring publicly, but will mention
your name to those who run this process to make sure you didn't slip between the cracks.